examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -62,30 +62,29 @@
                 add_eeg_ref=False, verbose=False)
 
 # Create classification pipeline
-clf = make_pipeline(Xdawn(n_components=3),
-                    EpochsVectorizer(),
+X = EpochsVectorizer().fit_transform(epochs)
+labels = epochs.events[:, -1]
+clf = make_pipeline(XdawnTransformer(n_components=3,
+                    n_chan=epochs.info['nchan']),
```

Comment:
It looks fine to mne. Running `pep8 examples/decoding/plot_decoding_xdawn_eeg.py` seems fine too.